code/modules/shuttle/shuttle_manipulator.dm

```diff
@@ -155,7 +155,7 @@
 					M.setTimer(50)
 					message_admins("[key_name_admin(usr)] fast travelled [M]")
 					log_admin("[key_name(usr)] fast travelled [M]")
-					feedback_add_details("shuttle_fasttravel", M.name)
+					SSblackbox.record_feedback("text", "shuttle_fasttravel", 1, "[M.name]")
```

Comment:
Wait, this is confusing. Sometimes (like here) you add extra quoting to have `"[M.name]"`, yet other times (e.g. in `code/modules/reagents/chemistry/holder.dm`) you REMOVE the quoting to have just
`SSblackbox.record_feedback("tally", "chemical_reaction", C.result_amount * multiplier, C.result)` (note the bare `C.result`).

Which one is it?

Reply:
Sometimes these things can be null, and `json_encode` will treat nulls as actual nulls with `C.result`, however, with `"[M.name]"`, if `M.name` is null, the response will still be `""`, which is not a null entry in `json_encode`